examples/plot_slicer_demo.py

```diff
@@ -50,6 +50,7 @@
 # 1.1 Default options
 # ^^^^^^^^^^^^^^^^^^^
 
+fig = plt.figure(figsize=(12, 12))
```

Comment:
This line is not needed. It creates an additional figure in the example. If you needed to use a custom figure size, you could pass it to the `plot_3d_slicer` e.g. `mesh.plot_3d_slicer(Lpout, fig=fig)`, but for the demo, I think it is simplest if we stick with the defaults and let `mesh.plot_3d_slicer` generate the figure
```suggestion
```

![image](https://user-images.githubusercontent.com/6361812/57741932-c9c5a280-7673-11e9-8f34-7e6bcb5b2899.png)

Reply:
:wave: @dccowan: do you have any concerns if I commit the change suggested above?

Reply:
I decided to set the figure size because some of the text was overlapping. I removed the line a regenerated and for all intents and purposes it's actually not bad.